samples/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,115 @@
+.. _suit_flash_companion:
+
+SUIT flash companion
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The flash companion application allows the secure domain firmware to access the external memory during the :ref:`Software Updates for Internet of Things (SUIT) <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+Requirements
+************
+
+The application supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk/nrf54h20/cpuapp
+
+Overview
+********
+
+The flash companion implements a device driver for the external flash device and provides an IPC service to the secure domain.
+The secure domain firmware can request from the flash companion to read, erase, or write data to the external memory.
```

Comment:
Should it be flash companion **sample** or **firmware** in place of flash companion only? If so, then please check throughout the document.